rack-protection/lib/rack/protection/http_origin.rb

```diff
@@ -17,7 +17,7 @@ module Protection
     #
     # The `:allow_if` option can also be set to a proc to use custom allow/deny logic.
     class HttpOrigin < Base
-      DEFAULT_PORTS = { 'http' => 80, 'https' => 443, 'coffee' => 80 }
+      DEFAULT_PORTS = { 'http' => 80, 'https' => 443 }
```

Comment:
I think we can leave `'coffee'` here, it's silly, but it's not really a coffeescript thing  😄 

https://datatracker.ietf.org/doc/html/rfc2324

https://github.com/rack/rack/blob/1d71f8741bb8343711924505a2bbc82159667691/lib/rack/request.rb#L168

Reply:
🤦 I assumed too much, TIL!